src/ubuntu/22.04/cross/arm-alpine/Dockerfile

```diff
@@ -1,4 +1,15 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04-crossdeps-local AS builder
+ARG ROOTFS_DIR=/crossrootfs/arm
```

Comment:
I think you can define the arg outside a stage scope to have a single shared default value.  You sill have to declare the ARG in each stage it is used.  This will avoid having "/crossrootfs/arm" defined in multiple locations.

Example

ARG ROOTFS_DIR=/crossrootfs/arm

FROM stageA
ARG ROOTFS_DIR

FROM stageB
ARG ROOTFS_DIR

Reply:
As we follow this pattern in a number of images, I'd like to do the refactoring in a follow-up PR (so as to hit all of the cases, including the cbl-mariner and azurelinux images). Is that okay with you?

Reply:
Yes that is fine.